  data_processing.cpp
LINK : fatal error LNK1104: nie można otworzyć pliku „python37_d.lib”
